index.js

```diff
@@ -334,7 +341,8 @@ module.exports = (command, args, options) => {
 			killed: false,
 			signal: null,
 			cmd: joinedCommand,
-			timedOut: false
+			timedOut: false,
+			canceled: false
```

Comment:
I would prefer to name it `isCanceled`. I realize it's inconsistent with `killed`, but I want to rename that one at some point too. Use the `isCanceled` naming for all the `canceled` variables.